### DISCUSSION TITLE
Different versions of same domain.

### DISCUSSION THREAD

**Discussion by ghost** [Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist#624] [Q&A]
Many major sites are still accessible with domains and numbers.
example,
https://xhamster16.desi/

The internet is for porn.

**Comment by spirillen** [comment]
You should report any phonographic domains to https://mypdns.org/my-privacy-dns/porn-records

Just follow the templates and your submission is added rather fast.

PS: the domain above is added here => https://mypdns.org/my-privacy-dns/porn-records/-/issues/831
